image_data_guide/04c_pytorch_training.ipynb

```diff
@@ -0,0 +1,1126 @@
xGa13G4ylp7rG23kHXdHw/xfN8wlBFUXR8L0fXVVRJZbTRQ5Elfv7nf5bf/u3f5vz8nI2NDfI85YP372HbJlQGUeSvkJgLFMnC1mvSAPIYpp6HaNBVKy57SWkoq89C+CgMQyfPc/Iig6pE02WobaqqElpspUbXNYq8QJHlleFK6FJFR21FW0tTFFlEbeV5vsLHFnieh+ggViAVaLpEWeYglcTJFx/wf1FLqUsUCupaRq4VKmR63SEvf/UlFLPieHnC2tqIfm/ED+//fa63r+C6LhuRxYenPyHwFkjSNooiidhcCZBEt7GqM1odlyTNkWSdslbJZR/dNqnqmsnkgA8++l3WBjbjyT5VFdNvX+fuvQ/JzYosrZBkk/EiYrzwWPgBqq6QZjl23SHxQ6p4zsXZe7Q7BffuGwThLdY371CVBf50yvz8lO3RgCqLKRUZVYHrV2/Q722yv3dC4H8ivnfpEt3MqeoYTYc4S6kqg2azieM4mC0IPm1gKw6WEmJqPpGsMV5UGEaJ0zFJPImLC49Op40f+8wvDpluNknzJVs7PebzMVk1RK0D3ny1j1RXLBYe2ztr2AaML+YcHZ+SZCqLMEKSDYpKQsoK6sNT/sHv/A672+v8m9/+aX70/ts4lkgv+OMPDnh4mmJqMrap0pYkkqWP1tkkTqCuFSazMc9df57f+I3fpMhyvvG1b/A3/sZ/xAfvvMXBwQH7xyd8wzH5bO8hf/nf+w6/WHyb//a//7v86MfvkNZ/Cs2pVNVMF1Nh+88T3KbD0dEBo+GrjM8vyGOfMk/56Cfvs7m5SewHmKbJ/uNHhEHM7u5VHMNgOplx7J/z3K2b/O73vs/Na9e4dfM6smKjmhJhGKIaCpPDcxoNE11XOTk5QtM05vM5rtvEsmziOGUynXD16lUenx3x8NEe0uPH9Efr/PAn77B14ypeEPFv//K/S10VVHnAxcWpyFVzW3gLn7ZrstkVuNIoilhfXxdu37Ikz3M0TSNOU1qdNkE6o65qeq02n37yCTIStmHy8YcfEQQBn92/z82bN1nvD+m027QaLoYstCsff/IJVstluZwzbHepJRO759LrtlHkGtfaQjMUojShM+jxwu0XeP8Tj1vXdqmrAlkRnerNzW0yasLFnDRKMU2Ffq+HhszDT+5x5+UXuXn1Oj/4Z7+Ppmm8+PJLHO6fc+3KlZVJRYyn6rqm3W6zv7/PaDRimXiEnk+3PxJULz9ic7QpoiN0lSAK2dzcJAgCdnZ28LwF52dnXLm2S1rktJoNdFX90o6VAOIsoWEppFnCfCnME4ZtcX50gaKIkVir1eLajVvUZc325i6dTpuqLoTru645Pz2n5TbEiLqo0XRTZN/GEWEY0rQdTN1gcnbBKy+9TBaFbG6uc3JydsnKtiwLpJo0FQWXoenkhhgVjvoD8qq+5ISnaUqRCo31s7gYRdbQNNGVUmQN26p48uQJjXZ7ZQyZYepitKqrgsakyhpFUbK3d0Cj6eC6LhfzCd1Oh2tXrvLw00dYqkWeFjQ7TeYLD7fV4vjkmGvXrnB6tE+r1cNbpMzmPo7pMB5P2d29ShAEREFMXQpjSK/Xpq4qTs7PWJM30XSd3atXOT49xbIcbLvB/pN9tHWd4+NTmo0W125s0O70+Ml7H3J+eoGhyDSdBouLMUgZo+FN6ipDV+H27RtEUSRSBnpd6rrGaTSQFTFdKIqCMPPJM4WG7VAi3vPlPCTwvZVW1qCuYGtzC9u2efr0KWujLknw5e08pWkhXPMllGVNJVXC3GOKTpFhaCJAW1FYLGaomoGm61RVRhB4YvTnWNR1jWnYKIWJokicnc3Z23uXl16+xbWrL/P48VNms4SNjXXuvPQV/ugP30LXasYXS7xlzMXFBd4yRkJne9PkYuIjSSEScHBwgGEYGIa2YmanYrReZeiGSZ6nuK5OVReYlsTZ+SFrG02gYme7z3KmsFwI5jjk1Ki0ugYDrXMZz37mIBIAACAASURBVBN4JXZDZjjqYFqiU1/XonOb5wWO3UKSCpIkoVzhKL15TLvd5uLsgrffegfHaXIxveDiYkJdl6iKTpJkFMUCy3SoKsR0wRBGk6qCIEhWCFWQUEAR5rCs/Bx/qSjKKpZIxjAMFEXB9wMsU1uFxitUVUmaxSJ2q1Q+j3CTy9X3XoS0y7KyQpeCruuXZCpgJSUQOdQCd6kgSQjT1Jd2icuUJElQgyxLrA1HBMEuuq7z0ksvMRpu8fTRU9745jdpNGz+QfY7vPUHPyBNCkHR0xWQRBasLD17PVHYn5we03AtOp0OfjBjNrtAU2VmM4/FxMdUBSXMNCuQcjrdFotpQJZWlFVFEHg0bIuy7QoSWZ4yj2KSRODPd3a2SNKQT+99xGweEIYlFSp/9IPv8+677+C6Lp1eh/liTJysk+UBTsNC1UouLi7Isozlcrkin6lkWUZRiEvc+mhDyMPU6PK5b1k9ZFnIZEajERcXexCWJBEEQYBtC9x0EASMxxN0s6TZ6qAbBj/68V2++sodFEVmY30kLpg5TKYh05nHwaFHXarUNRRlTVHVSLXEfBGQhgHHx8fMJwve/IU3KGuLbnfEk0fHlIXCIsqIwwpdMmi6LtNFynI5p5YlptMxX/2aTFnpJHnNO+/dIyv/Hn/9r/0SL73wCvc+u4chaZzuHSPXOmUNf/tv/qf8w9/7R/yX/82v/Wvsnj9h1XVJXoq4pkoS+iNx2Ao8aKfVpt/rIQGPHj5kfbRGnmY0bAfbtAj9gLOTUy7OBUoxixOyLGW5XHJwcMDJ8bHgKTcaK/JLiaxIVJX48idJgqKIzud0OqeqKi6Wc5K6ZB76jJdT7j68x/7ZCbpj4GcRqqny6f17/OS997gYjzk8PgAq4iQiigJMQ4UiJssy5tOZYHiv4kA6nQ6yLET3QRDgL5fEYcj4/IKzk1PiIGQ8FkjTF55/njt37jAajUiThCub29RZwcP7n2FrBq+++BK6rJD4IVJRsbU54tWXX2
```

Comment:
Nice notebook. You can add a small blurb at the end describing what other notebook options they have for section 4].

Reply:
added a note to all 03 and 04 series notebooks about the other tracks available to run